kong/templates/nginx_kong_stream.lua

```diff
@@ -34,6 +34,10 @@ $(el.name) $(el.value);
 > end
 
 init_by_lua_block {
+> if test_coverage then
```

Comment:
Perhaps?

```suggestion
> if test and coverage then
```

But `test_coverage` is fine too.